src/services/refactors/extractType.ts

```diff
@@ -157,7 +157,7 @@ namespace ts.refactor {
                     return true;
                 }
             }
-            else if (isTypeQueryNode(node)) {
+            else if (isTypeQueryNode(node) && node.exprName.kind !== SyntaxKind.ClassExpression) {

```

Comment:
Just wondering out loud: it might be useful to make an `isTypeQueryableNode` helper or similar, since this is already used twice?